### PR TITLE
Adaptation of Modal View

### LINKED ISSUE #190: Frontend: Adaptation of Modal View Component with recent changes
## **User story** 
**Who:** As a developer
**What:** I want to use real data from database and add the new components in the Modal View Component.
**Why:** So that modal will be able to show the property values of objects that is sent from recent database.
## **Acceptance criteria**

Modal View Component was using a dummy data which doesn't have the same property names of members table in database. 

Additionally, the close button was self created and the modal doesn't fit well with recent width and height of the pages. 

Therefore, the component should be modified considering these.

### PR DESCRIPTION
**Pull request by Rasim-Deniz**
# Description

Property names that Modal View Component was using changed related to data in database. Self-created close button and dropdown parts of the code is replaced with proper components.

Fixes #190 

# How to test?

Run npm run storybook.

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [ ] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [ ] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/components/ModalViewComponent/AdminPanel.js [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
-export default function AdminPanel() {
+export default function AdminPanel({ owner }) {
   return (
     <div className="modal-content">
       <div className="modal-body">
         <img
           className="round"
-          src="https://randomuser.me/api/portraits/men/79.jpg"
-          alt="member"
+          src={require(`../../assets/images/demo-members-photos${owner.photoUrl}`)}
```
Comment (senner007) [inline src/client/components/ModalViewComponent/AdminPanel.js:10]: I think the owner image will come from the members request also, right?

File: src/client/components/ModalViewComponent/membersData.json [hunk 2]
```diff
@@ -1,32 +1,30 @@
...
-  },
-  {
-    "id": 5,
-    "photo": "https://randomuser.me/api/portraits/men/83.jpg",
-    "name": "Arlene McCoy",
-    "role": "Member"
+    "userId": 4,
+    "role": "basic",
+    "fullName": "Dianne Russell",
+    "firebaseUId": "supersecret4",
```
Comment (senner007) [inline src/client/components/ModalViewComponent/membersData.json:27]: We most likely won't return the firebaseuids. No need for that?

**Review by senner007 — CHANGES_REQUESTED** [review]

File: src/client/components/ModalViewComponent/AdminPanel.js (hunk 1 above)
Comment (Rasim-Deniz) [reply src/client/components/ModalViewComponent/AdminPanel.js:10]: Yes, I thought that way by maybe adding photoUrl in select query of that get all members endpoint.

File: src/client/components/ModalViewComponent/membersData.json (hunk 2 above)
Comment (Rasim-Deniz) [reply src/client/components/ModalViewComponent/membersData.json:27]: Yeah, there was no need here. I just added them if they changed the way of getting photos like retrieving from firebase maybe?

File: src/client/components/ModalViewComponent/AdminPanel.js (hunk 1 above)
Comment (Rasim-Deniz) [reply src/client/components/ModalViewComponent/AdminPanel.js:10]: @senner007 Lint is not accepting use of require here. How can I modify it to get rid of it?

File: src/client/components/ModalViewComponent/membersData.json (hunk 2 above)
Comment (Rasim-Deniz) [reply src/client/components/ModalViewComponent/membersData.json:27]: So are we not gonna use firebaseUId? Then I will delete this.

Comment (senner007) [reply src/client/components/ModalViewComponent/membersData.json:27]: yes

**Comment by senner007** [comment]
Looks good. Is it done?

**Review by senner007 — APPROVED** [review]
✔

File: src/client/components/ModalViewComponent/AdminPanel.js (hunk 1 above)
Comment (Rasim-Deniz) [reply src/client/components/ModalViewComponent/AdminPanel.js:10]: I have no idea how I'm gonna solve this lint error. Googled it but they are not working.

Comment (senner007) [reply src/client/components/ModalViewComponent/AdminPanel.js:10]: Should this url not come from the databse? The photo url?

Comment (Rasim-Deniz) [reply src/client/components/ModalViewComponent/AdminPanel.js:10]: Yes

Comment (Rasim-Deniz) [reply src/client/components/ModalViewComponent/AdminPanel.js:10]: without using require it is not working but when I add required lint errors come up

Comment (senner007) [reply src/client/components/ModalViewComponent/AdminPanel.js:10]: Again, I don't see the need to use require here. Only owner.photoUrl

Comment (Rasim-Deniz) [reply src/client/components/ModalViewComponent/AdminPanel.js:10]: When I use this `src={`../../assets/images/demo-members-photos${member.photoUrl}`}` it looks like : 
image

Comment (Rasim-Deniz) [reply src/client/components/ModalViewComponent/AdminPanel.js:10]: Even if I use a json file like this : 
```
[
  {
    "userId": 1,
    "role": "basic",
    "fullName": "Darrell Steward",
    "photoUrl": "../../assets/images/demo-members-photos/member1.jpg"
  },
  {
    "userId": 2,
    "role": "basic",
    "fullName": "Kristin Watson",
    "photoUrl": "../../assets/images/demo-members-photos/member2.png"
  },
  {
    "userId": 3,
    "role": "owner",
    "fullName": "Theresa Webb",
    "photoUrl": "../../assets/images/demo-members-photos/member3.png"
  },
  {
    "userId": 4,
    "role": "basic",
    "fullName": "Dianne Russell",
    "photoUrl": "../../assets/images/demo-members-photos/member4.png"
  }
]
```
and use this `member` in Member component, it doesn't work...

Comment (senner007) [reply src/client/components/ModalViewComponent/AdminPanel.js:10]: Can't you just hardcode to a real url in the json file for now?

Comment (Rasim-Deniz) [reply src/client/components/ModalViewComponent/AdminPanel.js:10]: Like how? What do you mean by real url?

Comment (senner007) [reply src/client/components/ModalViewComponent/AdminPanel.js:10]: online photos

Comment (Rasim-Deniz) [reply src/client/components/ModalViewComponent/AdminPanel.js:10]: Then is this gonna be changed later related to database?

**Review by senner007 — APPROVED** [review]

File: src/client/components/ModalViewComponent/AdminPanel.js (hunk 1 above)
Comment (Rasim-Deniz) [reply src/client/components/ModalViewComponent/AdminPanel.js]: @senner007 Can I have an answer here please?

**Review by shanhaider23 — APPROVED** [review]

File: src/client/components/ModalViewComponent/AdminPanel.js (hunk 1 above)
Comment (senner007) [reply src/client/components/ModalViewComponent/AdminPanel.js]: You will get the photo urls from the users from the database right. What you are using now is just a fake json to  display in storybook

Comment (Rasim-Deniz) [reply src/client/components/ModalViewComponent/AdminPanel.js]: So if I use `src={owner.photoUrl}` only, are we gonna be able to retrieve images when the real data is sent from db?

Comment (Rasim-Deniz) [reply src/client/components/ModalViewComponent/AdminPanel.js]: I won't actually, I wanted to adapt it related to real database but it won't work.

Comment (senner007) [reply src/client/components/ModalViewComponent/AdminPanel.js]: When the use is created in our database we upload the photo url from the firebase data in the browser. Therefore each user will have a column with their url pointing to their photo

Comment (Rasim-Deniz) [reply src/client/components/ModalViewComponent/AdminPanel.js]: Oh ok. Then should I leave like I did in last commit?

Comment (senner007) [reply src/client/components/ModalViewComponent/AdminPanel.js]: No lets merge it. Then you can see how it works later

**Pull request merged**